src/distributions/poisson.rs

```diff
@@ -108,29 +108,35 @@ mod test {
     use distributions::Distribution;
     use super::Poisson;
 
+    #[cfg(not(miri))] // Miri is too slow
+    const N: u32 = 1000;
+    #[cfg(miri)]
+    const N: u32 = 100;
```

Comment:
In this case the count is used to calculate an average, and reducing the number of samples also increases the distribution. IMO it makes more sense just to avoid running these tests (you already disabled one of them).